### PR TITLE
UBERF-5827: add collaborative description for companies

### PR DESCRIPTION
**Pull request by kristina-fefelova**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does a local formatting is applied (rush format)
* [ ] - Does a local svelte-check is applied (rush svelte-check)
* [ ] - Does a local UI tests are executed [UI Testing](../tests/readme.md)
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

## Contributor requirements

* [ ] - Sign-off is provided. [DCO](https://github.com/apps/dco)
* [ ] - GPG Signature is provided. [GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)

<sub>View in Huly <a href="https://front.hc.engineering/guest/platform?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NWUxZjUzMzExOWU2NDMwMGNhYTc3OWMiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InBsYXRmb3JtIiwicHJvZHVjdElkIjoiIn0.5grWSCynCjKX61jXjuXTsniqantvaibKTTlq8sAg8-A">UBERF-5849</a></sub>

### CODE REVIEW

**Comment by kristina-fefelova** [comment]
Screenshot from 2024-03-01 19-26-53

File: plugins/recruit-resources/src/utils.ts [hunk 1]
```diff
@@ -72,7 +72,7 @@ async function generateIdLocation (loc: Location, shortLink: string): Promise<Re
   }
   const appComponent = loc.path[0] ?? ''
   const workspace = loc.path[1] ?? ''
-  const objectPanel = hierarchy.classHierarchyMixin(recruit.mixin.Candidate as Ref<Class<Doc>>, view.mixin.ObjectPanel)
+  const objectPanel = hierarchy.classHierarchyMixin(doc._class, view.mixin.ObjectPanel)
```
Comment (haiodo) [inline plugins/recruit-resources/src/utils.ts]: I suppose you need Hierarchy.mixinOrClass, please check.

Comment (kristina-fefelova) [reply plugins/recruit-resources/src/utils.ts]: Done.

**Comment by kristina-fefelova** [comment]
Added description into create modal:
Screenshot from 2024-03-04 12-41-11
Screenshot from 2024-03-04 12-41-36
Screenshot from 2024-03-04 12-42-38

**Review by ThetaDR — APPROVED** [review]

**Review by haiodo — APPROVED** [review]

**Pull request merged**